### ISSUE TITLE
Files in wheel have wrong permissions, making it impossible to import the package

### ISSUE THREAD

**Issue by ishirav** [pinax/django-mailer#96]
After installing the package using buildout, it is not possible to import anything from it unless running as root. This is caused by faulty permissions in the wheel file: as you can see below, only root has read permissions to the files. The correct mode should be `-rw-rw-r--`

```
-rw-r----- 1 root root 1.3K Jan 11 09:56 admin.py
-rw-r----- 1 root root 2.5K Jan 11 09:56 admin.pyc
-rw-r----- 1 root root  367 Jan 11 09:56 backend.py
-rw-r----- 1 root root 6.8K Jan 11 09:56 engine.py
-rw-r----- 1 root root 3.8K Jan 11 09:56 __init__.py
-rw-r----- 1 root root 4.2K Jan 11 09:56 __init__.pyc
drwxr-xr-x 3 root root 4.0K Jan 11 09:56 locale
drwxr-xr-x 3 root root 4.0K Jan 11 09:56 management
drwxr-xr-x 2 root root 4.0K Jan 11 09:56 migrations
-rw-r----- 1 root root 8.8K Jan 11 09:56 models.py
-rw-r----- 1 root root  13K Jan 11 09:56 models.pyc
drwxr-xr-x 2 root root 4.0K Jan 11 09:56 south_migrations
-rw-r----- 1 root root  29K Jan 11 09:56 tests.py
```

It seems like pip and easy_install fix the permissions, but buildout doesn't.

**Comment by wiggin15** [comment]
To be clear, this seems to be a problem with the wheel distribution only. You can see the permissions are wrong if you unzip the wheel file from pypi.
buildout (the `buildout.wheel` extension to be exact) simply uses the "wheel" package for installation. The following lines will also install the wheel file with incorrect permissions:

```python
import wheel.install
wheel.install.WheelFile("django_mailer-1.2.3-py2-none-any.whl").install()
```
The impact is that if you install the package with `sudo`, only `root` can access the installed package.

**Comment by ishirav** [comment]
FWIW, the problem is still present in v1.2.4

**Comment by spookylukey** [comment]
Should be fixed in v1.2.6
